exonum-java-binding-qa-service/src/main/java/com/exonum/binding/qaservice/transactions/UnknownTx.java

```diff
@@ -34,32 +32,24 @@
  * once they receive it as a message from this node. If multiple unknown transaction messages
  * need to be submitted, a seed might be added.
  */
-public final class UnknownTx extends AbstractTransaction {
+public final class UnknownTx implements Transaction {
 
-  static final short ID = 9999;
-
-  public UnknownTx() {
-    super(createMessage());
-  }
-
-  @Override
-  public boolean isValid() {
-    return true;
-  }
+  private static final short ID = 9999;
 
   @Override
-  public void execute(Fork view) {
+  public void execute(TransactionContext context) {
     throw new AssertionError("Must never be executed by the framework: " + this);
   }
 
-  private static BinaryMessage createMessage() {
-    return new Message.Builder()
-        .setServiceId(QaService.ID)
-        .setMessageType(ID)
-        .setNetworkId((byte) 0)
-        .setVersion((byte) 0)
-        .setBody(ByteBuffer.allocate(0))
-        .setSignature(new byte[Message.SIGNATURE_SIZE])
-        .buildRaw();
+  /**
+   * Returns raw transaction.
+   */
+  public static RawTransaction createRawTransaction() {
+    return RawTransaction.newBuilder()
+        .serviceId(QaService.ID)
+        .transactionId(ID)
+        .payload(new byte[]{})
```

Comment:
Shall we use empty payload as the default in RawTransaction.Builder?
        Probably, not, because usually you can't have a tx with empty body (need some kind of
         a seed to distinguish tx instances from the same author).